src/main/java/com/openshift/restclient/api/capabilities/IPodExec.java

```diff
@@ -0,0 +1,150 @@
+/******************************************************************************* 
+ * Copyright (c) 2016 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
+package com.openshift.restclient.api.capabilities;
+
+import com.openshift.restclient.capability.ICapability;
+import com.openshift.restclient.capability.IStoppable;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Runs container exec
+ */
+public interface IPodExec extends ICapability{
+
+	/**
+	 * Execute a command on a named container in this pod
+	 * @param listener Listener for command output
+	 * @param options Options for the exec
+	 * @param commands A command to run and any arguments
+	 * @return A Handle to allow termination of the connection
+	 */
+	IStoppable start( IPodExecOutputListener listener, Options options, String... commands );
```

Comment:
Fair point - there is precedence.  The fact that I don't agree with it is another thing ;-) but sure, stay consistent with JSE.